packages/modal/src/components/BaseModal.tsx

```diff
@@ -1,139 +1,44 @@
   draggable?: boolean;
 }
 
 export const DRAGGABLE_HANDLE_CLASSNAME = "draggable-modal-handle";
 
 export const BaseModal: React.FC<BaseModalProps> = ({
-  width = "900px",
-  background = "primaryBg",
+  width,
   draggable = false,
   children,
   ...props
 }) => {
-  const { colors } = useThemeFields(
-    {
-      colors: {
-        background
-      }
-    },
-    [background]
-  );
-  const { modalShadow } = useThemeSelector(
-    theme => ({
-      modalShadow: theme.shadows.modal
-    }),
-    []
-  );
-
   return (
-    <ClassNames>
-      {({ css, cx }) => {
-        const contentClassName = css`
-          position: relative;
-          display: block;
-
-          background: ${colors.background};
-          box-shadow: ${modalShadow};
-          pointer-events: all;
-
-          width: ${width};
-          max-width: 100%;
-
-          animation: ${modalAnimateIn} 0.3s cubic-bezier(0.645, 0.045, 0.355, 1)
-            both;
-
-          @media print {
-            box-shadow: none;
-            top: 0;
-          }
-
-          :focus {
-            outline: 0;
-          }
-        `;
-
-        const handle = css`
-          .${DRAGGABLE_HANDLE_CLASSNAME} {
-            cursor: move;
-          }
-        `;
-
-        return (
-          <ReactModal
-            overlayClassName={css`
-              position: fixed;
-              z-index: 10;
-              top: 0;
-              left: 0;
-              right: 0;
-              bottom: 0;
-              background-color: rgba(0, 0, 0, 0.5);
-              overflow: auto;
-              display: flex;
-              flex-direction: column;
-              justify-content: flex-start;
-              align-items: center;
-              padding: 8px;
-
-              animation: ${fadeIn} 0.3s cubic-bezier(0.645, 0.045, 0.355, 1)
-                both;
-
-              @media print {
-                background-color: rgba(255, 255, 255, 1);
-              }
-            `}
-            className={css`
-              max-width: 100%;
-              outline: none;
-              pointer-events: none;
-            `}
-            {...props}
-          >
-            <Draggable
-              handle={`.${DRAGGABLE_HANDLE_CLASSNAME}`}
-              bounds=".ReactModal__Overlay"
-              disabled={!draggable}
-            >
-              <div className={cx(contentClassName, { [handle]: draggable })}>
-                {children}
-              </div>
-            </Draggable>
-          </ReactModal>
-        );
-      }}
-    </ClassNames>
+    <ReactModal
+      overlayClassName={styles.Overlay}
+      className={styles.Modal}
+      {...props}
+    >
+      <Draggable
+        handle={`.${DRAGGABLE_HANDLE_CLASSNAME}`}
+        bounds=".ReactModal__Overlay"
+        disabled={!draggable}
+      >
+        <div
+          style={{ "--swui-modal-width": width } as any}
+          className={cs(styles.Content, {
+            [styles.isDraggable]: draggable
```

Comment:
Do you need to do a `const cx = classnames.bind(styles);` for this to work?

Reply:
No you don't :)

Reply:
Resolved then I guess?